src/main/kotlin/org/malachite/estella/commons/loader/FakeUsers.kt

```diff
@@ -32,7 +32,11 @@ object FakeUsers {
             mail = "pecunia@non.olet",
             password = "a"
         )
+    )
 
+    val organizationUsers = listOf(
+        User(null,"Qualtrics","","hr@qualtrics.com","a"),
```

Comment:
Mamy nazwę firmy i tutaj i w firmie - nie jest to trochę redundancja?

Reply:
Można usunąć

Reply:
szczerze Ci powiem, że nie mam pojęcia XD

Reply:
Wydaje mi się, że najlepiej byłoby po prostu puste pole zostawić zamiast imienia jako qualtrics